src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketPREnvironmentContributor.java

```diff
@@ -0,0 +1,34 @@
+package com.atlassian.bitbucket.jenkins.internal.scm;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.EnvVars;
+import hudson.Extension;
+import hudson.model.*;
+import jenkins.branch.Branch;
+import jenkins.branch.BranchProjectFactory;
+import jenkins.branch.MultiBranchProject;
+import jenkins.scm.api.SCMHead;
+
+@Extension
+public class BitbucketPREnvironmentContributor extends EnvironmentContributor {
```

Comment:
This is the the extension we will need for [JENKINS-60340](https://issues.jenkins.io/browse/JENKINS-60340), so when we put that in review this will probably become the more general `BitbucketEnvironmentContributor`